api/src/test/java/com/epam/pipeline/test/creator/cluster/NodeCreatorUtils.java

```diff
@@ -86,4 +99,20 @@ public static AllowedInstanceAndPriceTypes getDefaultAllowedInstanceAndPriceType
     public static NodeDisk getDefaultNodeDisk() {
         return new NodeDisk(ID, TEST_STRING, LDT);
     }
+
+    public static PipelineRun getPipelineRun() {
```

Comment:
Let's not confuse packages: `PipelineRun` is not in cluster package. We need to create a new package/class for `PipelineRun` object creating methods.

Reply:
The same for creators below